Detections/AzureDevOps/AzDOServiceConnectionUsage.yaml

```diff
@@ -0,0 +1,32 @@
+id: d564ff12-8f53-41b8-8649-44f76b37b99f
+name: Azure DevOps Service Conection Abuse
+description: |
+  'This detection flags builds/releases that use a large number of service connections if they aren't manually allowlisted.
+  This is to determine if someone is hijacking a build/release and adding many service connections in order to abuse or dump credentials from service connections.'
+severity: High
+queryFrequency: 2h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
```

Comment:
Please use the MITRE ATT&CK full names with no spaces. Here Escalation would be PrivilegeEscalation, there is no Impact so will need to remove or change to another and we also require Techniques in Detections - https://attack.mitre.org/techniques/enterprise/

Format as follows -

relevantTechniques:

T1078
T1110

Reply:
fixed in next commit